### PR TITLE
Show message when module was added to dashboard

### PR DESCRIPTION
**Pull request by fancyFranci**
Pull Request for Issue # 

### Summary of Changes
When you add a module to the dashboard, nothing happens until you reload the page. Now at least a message shows up. The permissions were not correctly checked and an account with not enough permissions hat problems after the modal opened. That is fixed in this PR too.

### Testing Instructions
Go to the dashboard and click on 'add dashboard module'. After successfully doing that, a message appears. When you have not permissions to create or manage a module, you do not see this button.

### CODE REVIEW

File: administrator/components/com_cpanel/tmpl/cpanel/default.php [hunk 1]
```diff
@@ -34,9 +33,11 @@
 		'url'         => Route::_('index.php?option=com_cpanel&task=addModule&function=jSelectModuleType&position=' . $this->escape($this->position)),
 		'bodyHeight'  => '70',
 		'modalWidth'  => '80',
-		'footer'      => '<button type="button" class="button-cancel  btn btn-sm btn-danger" data-dismiss="modal" data-target="#closeBtn" aria-hidden="true"><span class="icon-cancel" aria-hidden="true"></span>'
+		'footer'      => '<button type="button" class="button-cancel  btn btn-sm btn-danger" data-dismiss="modal" data-target="#closeBtn" aria-hidden="true">'
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/cpanel/default.php:36]: why is the entire button aria-hidden?

File: administrator/language/en-GB/en-GB.com_cpanel.ini [hunk 2]
```diff
@@ -32,6 +31,7 @@ COM_CPANEL_MSG_HTACCESS_BODY="A change to the default .htaccess and web.config f
 COM_CPANEL_MSG_HTACCESS_TITLE=".htaccess & web.config Update"
 COM_CPANEL_MSG_LANGUAGEACCESS340_BODY="Since Joomla! 3.4.0 you may have issues with the System - Language Filter plugin on your website. To fix them please open the <a href=\"index.php?option=com_languages&view=languages\">Language Manager</a> and save each content language manually to make sure an Access level is saved."
 COM_CPANEL_MSG_LANGUAGEACCESS340_TITLE="You have possible issues with your multilingual settings"
+COM_CPANEL_MSG_MODULE_ADDED="Module successfully added to the dashboard! Please reload the page."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_cpanel.ini:34]: Please dont use the redundant word "successfully" or the exclamation mark

**Comment by brianteeman** [comment]
I really think this is the wrong approach. We should never be asking anyone to refresh the page

**Comment by bembelimen** [comment]
I have to agree with Brian, sorry. I'll close it....we need an automatical refresh.

**Pull request closed without merging**